inject-dist/observers.js

```diff
@@ -21,14 +22,25 @@ const OBSERVERS = [
   'document.write',
   'queueMicrotask',
   'requestIdleCallback',
-  'navigator.scheduling.isInputPending',
-  'scheduler.postTask'
+  'scheduler.postTask',
+  'window.matchMedia',
```

Comment:
`window` here might have been needed before the fix in #32. Should be safe to remove now. Also note that there's a merge conflict as a result of that PR. (`eval` was added to this list)

```suggestion
  'matchMedia',
```